packages/@react-spectrum/datepicker/src/DatePicker.tsx

```diff
@@ -64,6 +64,9 @@ export const DatePicker = React.forwardRef(function DatePicker<T extends DateVal
   let {direction} = useLocale();
   let domRef = useFocusManagerRef(ref);
   let stringFormatter = useLocalizedStringFormatter(intlMessages, '@react-spectrum/datepicker');
+  if (props.createCalendar) {
+    (calendarProps as SpectrumCalendarProps<DateValue>).createCalendar = props.createCalendar;
+  }
```

Comment:
Would we ever want to pass `createCalendar` into the `DateField` as well? If so we might need different props for them. In the 4-5-4 case, the behavior is pretty weird if you pass it into DateField. But in other cases, maybe someone would want that?